library/src/main/java/com/novoda/downloadmanager/lib/DownloadManager.java

```diff
@@ -644,7 +644,7 @@ public long addCompletedDownload(String title, String description,
         values.put(Downloads.Impl.COLUMN_MEDIA_SCANNED, (isMediaScannerScannable) ? Request.SCANNABLE_VALUE_YES : Request.SCANNABLE_VALUE_NO);
         values.put(
                 Downloads.Impl.COLUMN_VISIBILITY, (showNotification) ?
-                        Request.VISIBILITY_VISIBLE_NOTIFY_ONLY_COMPLETION : Request.VISIBILITY_HIDDEN);
+                        NotificationVisibility.ONLY_WHEN_COMPLETE : NotificationVisibility.HIDDEN);
```

Comment:
as in, the option to "not show a notification when downloading" probably won't work because we're not a system service so the service can be killed. This isn't something to change though, just a curiosity!